libraries/standard/http/src/private/http_client_internal.h

```diff
@@ -3,6 +3,11 @@
 
 #include "config.h"
 
+/* Defind the maximum header size for http_parser to keep track of for us. */
+#define HTTP_MAX_HEADER_SIZE    HTTP_MAX_RESPONSE_HEADERS_SIZE_BYTES
+
+#include "http_parser.h"
```

Comment:
Isn't this header included in the `http_client.c` source file? Is there any macro or type within this internal header file that depends on the `http_parser.h`?

Reply:
I took it out from http_client.c and in the next PR I will be putting the parsing context here that depends on struct http_parser.